python/py-scikit-image/Portfile

```diff
@@ -44,6 +44,16 @@ if {${name} ne ${subport}} {
         depends_run-append  port:py${python.version}-dask \
                             port:py${python.version}-toolz \
                             port:py${python.version}-cloudpickle
+    } elseif {${python.version} == 36} {
+        version             0.16.2
+        revision            0
+        epoch               1
```

Comment:
Why increase the epoch? It was not possible to install py36-scikit-image 0.17.x before because its dependency py36-tifffile does not exist.

Reply:
I increased the `epoch` because even though `py36-scikit-image @0.17.2_1` can't be installed, `py36-scikit-image @0.17.2_0` **was** able to install successfully since it did not specify a tifffile runtime dependency, as described in https://trac.macports.org/ticket/61178.